requirements.txt

```diff
@@ -18,4 +17,5 @@ pycodestyle==2.3.1        # via flake8
 pyflakes==1.5.0           # via flake8
 pygments==2.2.0
 pytest==3.1.2
+python-dateutil==2.6.0    # via faker
 six==1.10.0
```

Comment:
looks like you compiled the requirements.in on python 3, can you compile on python 2 please? enum34 and configparser are conditional on py2